### ISSUE TITLE
Editor appearing twice on dev.to/new

### ISSUE THREAD

**Issue by pkfrank** [forem/forem#7584]
**Describe the bug**

- I went to `/dashboard/`
- I clicked on `edit` of a previous post
- I copied the body
- I pressed "Write a post" at the top of the page
- I quickly copy/pasted the body from the previous post
- I scrolled down and realized that there were two editors on the page (one with the new text, one blank)
- I'll note I was moving pretty quickly doing this, and am using the v1 version of the editor

**To Reproduce**

I was able to reproduce this once using the above steps, but I haven't been able to do it consistently to determine why it sometimes happens.

**Screenshots**

![example of two editors](https://cl.ly/d22a5c061349/Image%202020-04-28%20at%207.33.20%20PM.png)

**Desktop (please complete the following information):**

- OS: Mac / Sierra 10.13
- Browser: Chrome
- Version: Version 81.0.4044.113 (Official Build) (64-bit)

**Comment by rhymes** [comment]
Thanks for the issue! We'll take your request into consideration and follow up if we decide to tackle this issue.

To our amazing contributors: issues labeled `type: bug` are always up for grabs, but for feature requests, please wait until we add a `ready for dev` before starting to work on it.

To claim an issue to work on, please leave a comment. If you've claimed the issue and need help, please ping @thepracticaldev/oss and we will follow up within 3 business days.

For full info on how to contribute, please check out our [contributors guide](https://github.com/thepracticaldev/dev.to/blob/master/CONTRIBUTING.md).

**Comment by lightalloy** [comment]
I think this is a duplicate of #6926. @pkfrank could you move your description there?

**Comment by danascheider** [comment]
@lightalloy I've been able to duplicate this issue using the steps @pkfrank listed, but haven't managed to repro the one in #6926 using the steps given there (or any other way). I found additionally:

1. If you click "Write Post" a second time while on the same page (when there are two editors visible), the second one goes away.
2. If you click "Write Post" a third time after that, then the whole screen is grey and I see only the header and footer of dev.to but not the body of the site.
3. If you click "Write Post" while editing another post, the editor that comes up for writing the new post has the same text as the existing post, which makes it really unclear if you're creating a new post or editing the existing one.

This is an issue I could work on if it's one the maintainers would like to see fixed.

**Comment by lightalloy** [comment]
@danascheider thanks for providing more details for this issue. It would be great to get this bug fixed, feel free to work on it.

**Comment by lightalloy** [comment]
@danascheider I've learned that a new editor version will be released soon which may resolve this and related bugs, so it makes sense to wait for the release and check if the issue remains.

**Comment by cmgorton** [comment]
@pkfrank has this issue occurred anymore? From the screenshots it looks like we have an updated editor now so want to check before closing this issue. Thanks.

**Comment by Ridhwana** [comment]
@cmgorton I'll go ahead and close this issue since the workflow has changed a bit since we implemented the new editor and I can no longer replicate it.
